src/quo2/components/community/discover_card.cljs

```diff
@@ -0,0 +1,134 @@
+(ns quo2.components.community.discover-card
```

Comment:
consider breaking these p.r's into smaller focused chunks in the future.

e.g a pr adding discover card component to quo2 library,
and another pr where you use the discover card component in the status app.